anaconda.spec.in

```diff
@@ -456,6 +458,10 @@ desktop-file-install --dir=%{buildroot}%{_datadir}/applications %{buildroot}%{_d
 %{_prefix}/libexec/anaconda/dd_*
 
 %changelog
```

Comment:
The change log entry is not necessary - the changelog entries are generated automatically, so the change will be noted in the next entry after this PR is merged. :)

Reply:
So should I add a new commit to this pull request in order to remove the change log entry?